test/test_gromov.py

```diff
@@ -145,3 +145,79 @@ def test_gromov_entropic_barycenter():
                                                 'kl_loss', 2e-3,
                                                 max_iter=100, tol=1e-3)
     np.testing.assert_allclose(Cb2.shape, (n_samples, n_samples))
+
+
+def test_fgw():
+    n_samples = 50  # nb samples
+
+    mu_s = np.array([0, 0])

```

Comment:
set the seed (look at documentation of make_2D_samples_gauss ) to make the test deterministic